.snapcraft/resources/preparenode

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+node_version="v8.11.3"
+
+unamem="$(uname -m)"
+if [[ $unamem == *aarch64* ]]; then
+        node_arch="arm64"
+elif [[ $unamem == *64* ]]; then
+        node_arch="x64"
+elif [[ $unamem == *86* ]]; then
+        node_arch="x86"
+elif [[ $unamem == *armv6l* ]]; then
+        node_arch="armv6l"
+elif [[ $unamem == *armv7l* ]]; then
+        node_arch="armv7l"
+else
+        echo "Aborted, unsupported or unknown architecture: $unamem"
+        return 2
+fi
+
+
+wget https://nodejs.org/dist/$node_version/node-$node_version-linux-$node_arch.tar.xz; tar xvf node-$node_version-linux-$node_arch.tar.xz --strip 1 > /dev/null;
```

Comment:
any reason to no split this into two lines?

same thing here, just removing `v` option from `tar` will make it silent, no need to `> /dev/null`

Reply:
No reason.  Copied directly from the inline in the snapcraft.yaml